### PR TITLE
Sort discovery search results by start date

### PR DESCRIPTION
**Pull request by kluo**
First sorts by relevance score, then by start date.
@stvstnfrd

### CODE REVIEW

File: openedx/stanford/lms/envs/common.py [hunk 1]
```diff
@@ -115,6 +115,7 @@
 PROGRESS_SUCCESS_BUTTON_URL = 'http://<domain>/<path>/{course_id}'
 PROGRESS_SUCCESS_BUTTON_TEXT_OVERRIDE = None
 SEARCH_FILTER_GENERATOR = 'openedx.stanford.lms.lib.courseware_search.lms_filter_generator.TileSearchFilterGenerator'
+SEARCH_DISCOVERY_SORT_FIELDS = '_score:desc,start:desc'
```
Comment (stvstnfrd) [inline openedx/stanford/lms/envs/common.py]: What kind of values do we see for `_score`?
- integers?
- what kind of range?
- what's the median?
- etc..

Comment (kluo) [reply openedx/stanford/lms/envs/common.py]: @stvstnfrd It's a positive floating point number. Nitty gritty explained here https://www.compose.com/articles/how-scoring-works-in-elasticsearch/

File: requirements/edx/base.txt [hunk 2]
```diff
@@ -58,7 +58,7 @@ edx-oauth2-provider==1.2.0
 edx-opaque-keys==0.4.0
 edx-organizations==0.4.4
 edx-rest-api-client==1.7.1
-git+https://github.com/Stanford-Online/edx-search.git@e2aba4a30cf2dc6aa901799596161a59fa3e3baa#egg=edx-search==1.2.1
+git+https://github.com/Stanford-Online/edx-search.git@5c2321a10f454c1b50aca8e0f53ff26cff2ffa16#egg=edx-search==1.2.2
```
Comment (stvstnfrd) [inline requirements/edx/base.txt]: Does this SHA need updated?
Maybe point it at the merge commit once search lands?

Comment (stvstnfrd) [reply requirements/edx/base.txt]: Also, version is definitely outdated :)

File: openedx/stanford/lms/envs/common.py (hunk 1 above)
Comment (stvstnfrd) [inline openedx/stanford/lms/envs/common.py]: Move up to keep alpha sorting :)

**Pull request merged**